docs/samples/llmisvc/dp-ep/qwen3-nvidia/llm-inference-service-dp-ep-multi-deepseek-r1-pd-p-deepep-ht-d-deepep-ht-gpu-private.yaml

```diff
@@ -0,0 +1,583 @@
+          - name: main
+            image: quay.io/rhoai/odh-llm-d-inference-scheduler-rhel9:rhoai-2.25
+            args:
+              - -v=4 # Debug level
+              - --poolName
+              - "{{ ChildName .ObjectMeta.Name `-inference-pool` }}"
+              - --poolNamespace
+              - "{{ .ObjectMeta.Namespace }}"
+              - --zap-encoder
+              - json
+              - --grpcPort
+              - "9002"
+              - --grpcHealthPort
+              - "9003"
+              - --secureServing
+              - --modelServerMetricsScheme
+              - https
+              - --modelServerMetricsHttpsInsecureSkipVerify
+              - --certPath
+              - /etc/ssl/certs
+              - --configText
+              - |2
+              
+                apiVersion: inference.networking.x-k8s.io/v1alpha1
+                kind: EndpointPickerConfig
+                plugins:
+                - type: pd-profile-handler
+                  parameters:
+                    threshold: 0
+                - type: prefill-header-handler
+                - type: prefill-filter
+                - type: decode-filter
+                - type: prefix-cache-scorer
+                - type: load-aware-scorer
+                - type: max-score-picker
+                schedulingProfiles:
+                - name: prefill
+                  plugins:
+                  - pluginRef: prefill-filter
+                  - pluginRef: prefix-cache-scorer
+                    weight: 2.0
+                  - pluginRef: load-aware-scorer
+                    weight: 1.0
+                  - pluginRef: max-score-picker
+                - name: decode
+                  plugins:
+                  - pluginRef: decode-filter
+                  - pluginRef: prefix-cache-scorer
+                    weight: 2.0
+                  - pluginRef: load-aware-scorer
+                    weight: 1.0
+                  - pluginRef: max-score-picker
+    route: { }
+    gateway: { }
+  template:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/hostname
+                  operator: In
+                  values:
+                    - pokprod-b93r38s0
+                    - pokprod-b93r38s1
+                    - pokprod-b93r39s1
+                    - pokprod-b93r38s2
+                    - pokprod-b93r38s3
+    imagePullSecrets:
+      - name: "quay-registry-wseaton"
+    serviceAccountName: hfsa
+    containers:
+      - name: main
+        image: quay.io/aipcc/rhaiis/cuda-ubi9:3.2.2-1757699034
+        env:
+          - name: VLLM_LOGGING_LEVEL
+            value: INFO
+          - name: KSERVE_INFER_ROCE
+            value: "true"
+          - name: CUDA_DEVICE_ORDER
+            value: "PCI_BUS_ID"
+          # Memory optimizations
+#          - name: VLLM_MOE_DP_CHUNK_SIZE
+#            value: "32"
```

Comment:
Again purely from `docs` perspective - adding more context would help, e.g.: what is the default, and what does setting 32 change here?